### PR TITLE
Added features bundle

### PR DESCRIPTION
**Pull request by yannickl88**
Q | A
-- | --
License | MIT

### CODE REVIEW

**Comment by fabpot** [comment]
What about also adding a default config file as an example on how to use the bundle?

**Comment by yannickl88** [comment]
I can't really add much other then commented stuff since the bundle does not have any revolvers.

**Comment by sstok** [comment]
Service example filename is wrong, should be something like features (not your the Vendor name) 😉

**Comment by yannickl88** [comment]
@sstok you are right, my bad

**Comment by iltar** [comment]
@fabpot seems like https://pr3.contrib.symfony.sh/ is not working anymore: 
image

If I try: https://pr1.contrib.symfony.sh/ it "works", pr2 is down as well.

image

**Comment by stof** [comment]
@iltar the URL used by the server changed. It is now https://symfony.sh/r/contrib/3

As the PR has not been updated after the backend change, the URL reported to github is still the old one

**Comment by iltar** [comment]
Ah that works for me :+1:

File: yannickl88/features-bundle/1.0/etc/packages/features.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```
Comment (stof) [inline yannickl88/features-bundle/1.0/etc/packages/features.yaml]: @fabpot should we really have the service definition and the semantic configuration in the same file ? Shouldn't services be defined in a separate file ?

Comment (Pierstoval) [inline yannickl88/features-bundle/1.0/etc/packages/features.yaml]: In this case, the goal is to make sure services that depend on this package are removed when removing the package, else you would have to remove them manually.

It seems fine to me :/

Comment (iltar) [inline yannickl88/features-bundle/1.0/etc/packages/features.yaml]: The classes it refers to, do not exist, they are merely examples. I do agree with keeping them in here, I'm just not entirely sure if the examples should be in here.

Comment (e-moe) [inline yannickl88/features-bundle/1.0/etc/packages/features.yaml]: do we really need to have this file? see similar comment from @stof in sibling PR: https://github.com/symfony/recipes-contrib/pull/6#discussion_r113647743

**Comment by yannickl88** [comment]
Either way, this is just example config (and service definition) since the bundle does not provide any implementations. I added this on request of @fabpot (see first comment)

Anyhow, the bundle does not throw any errors when nothing is configured so everything is optional, hence why I commented everything.

Let me know if I need to remove the config file, change it or keep it as is.

**Comment by fabpot** [comment]
I think having commented configuration to help people get started seems like a good idea to me.

**Comment by yannickl88** [comment]
so... what can this be merged in it's current state or do I need to change anything?

File: yannickl88/features-bundle/1.0/etc/packages/features.yaml (hunk 1 above)
Comment (fabpot) [inline yannickl88/features-bundle/1.0/etc/packages/features.yaml]: These services should be removed, as they should be added by the dev in the main `container.yaml` file.

Comment (yannickl88) [inline yannickl88/features-bundle/1.0/etc/packages/features.yaml]: `container.yaml` is something that is part of the app? or is it something I need to add?

**Comment by Nyholm** [comment]
Thank you for this PR and sorry for the long wait. 
In order to get merged I need you to provide some good default config that will be good for 90% of the users.

**Comment by yannickl88** [comment]
@Nyholm That is the thing, I cannot add a good default since this bundle only adds infrastructure. The functional code has to be made by the person using this.

It's like asking for some default config for form component, you cannot add those without implementing a form type.

So at best I can add some commented service definitions, but that is about it.

See the [readme for an example](https://github.com/yannickl88/features-bundle).

**Comment by Nyholm** [comment]
I understand. Then I suggest closing this PR since Flex is smart enough to install this bundle without a recipe.

**Comment by Nyholm** [comment]
Im closing this PR because of inactivity. Feel free to open a new PR.

**Pull request closed without merging**